lib/r10k/source/git.rb

```diff
@@ -43,8 +43,9 @@ class R10K::Source::Git < R10K::Source::Base
   # @param basedir [String] The base directory where the generated environments will be created.
   # @param options [Hash] An additional set of options for this source.
   #
-  # @option options [Boolean] :prefix Whether to prefix the source name to the
-  #   environment directory names. Defaults to false.
+  # @option options [Boolean, String] :prefix A String value to prefix the environment directory name.
```

Comment:
uh, scratch that too :), I see the rest of the args are documented as well - i'll fix all three locations.